docs/RunningOnDevice.md

```diff
@@ -145,11 +145,11 @@ Type the following in your command prompt to install and launch your app on the
 $ react-native run-android
 ```
 
-> If you get a "bridge configuration isn't available" error, see [Using adb reverse](#using-adb-reverse).
+> If you get a "bridge configuration isn't available" error, see [Using adb reverse](docs/running-on-device.html#method-1-using-adb-reverse-recommended).
```

Comment:
Doing that seems to link directly to https://facebook.github.io/react-native/#method-1-using-adb-reverse-recommended (like the currently live site.)